app/controllers/graph_controller.rb

```diff
@@ -0,0 +1,14 @@
+class GraphController < ApplicationController
+
+  def index
+  end
+
+  def data
+    respond_to do |format|
+      format.json {
```

Comment:
preferably use `do...end` if it's a multi-line block  `{}` are for single line blocks https://github.com/bbatsov/ruby-style-guide#single-line-blocks